proposals/solid-oidc/index.bs

```diff
@@ -196,27 +196,108 @@ list.
 
 NOTE: the method by which the IdP resolves the WebID to an RDF document, is defined in
 [WebID 1.0](https://www.w3.org/2005/Incubator/webid/spec/identity/#processing-the-webid-profile).
-This example uses [Turtle](https://www.w3.org/TR/turtle/):
+This example uses [JSON-LD  ](https://www.w3.org/TR/json-ld11/):
 
 <div class='example'>
     <p>https://app.example/webid#id
 
     <pre>
-        @prefix solid: <http://www.w3.org/ns/solid/terms#> .
-
-        <#id> solid:oidcRegistration """{
-            "client_id" : "https://app.example/webid#id",
-            "redirect_uris" : ["https://app.example/callback"],
-            "client_name" : "Solid Application Name",
-            "client_uri" : "https://app.example/",
-            "logo_uri" : "https://app.example/logo.png",
-            "tos_uri" : "https://app.example/tos.html",
-            "scope" : "openid profile offline_access",
-            "grant_types" : ["refresh_token","authorization_code"],
-            "response_types" : ["code"],
-            "default_max_age" : 60000,
-            "require_auth_time" : true
-            }""" .
+        {
+          "@context": "https://www.w3.org/ns/solid/oidc-context.jsonld",
+
+          "client_id": "https://app.example/webid#id",
+          "client_name": "Solid Application Name",
+          "redirect_uris": ["https://app.example/callback"],
+          "client_uri": "https://app.example/",
+          "logo_uri" : "https://app.example/logo.png",
+          "tos_uri" : "https://app.example/tos.html",
+          "scope" : "openid profile offline_access",
+          "grant_types" : ["refresh_token","authorization_code"],
+          "response_types" : ["code"],
+          "default_max_age" : 3600,
+          "require_auth_time" : true
+        }
+    </pre>
+</div>
+
+### JSON-LD context
+
+This specification defines a JSON-LD context for use with OIDC client identifiers. This context is
+available at `https://www.w3.org/ns/solid/oidc.jsonld`. Client identifier documents that reference
+this JSON-LD context MUST use the HTTPS scheme.
+
+NOTE: the `oidc` vocabulary that is part of this context uses the HTTP scheme.
+
+The JSON-LD context is defined as:
+
+<div class='example'>
```

Comment:
Considering mentioning `http://www.w3.org/ns/solid/oidc` as the Solid OIDC vocabulary prior to the note.  Consider adding a document conventions/namespaces section and refer to it.

Examples (in this document) are informative. Consider clarifying what's the authoritative document defining the JSON-LD context.

Reply:
> Consider clarifying what's the authoritative document defining the JSON-LD context.

Since we require`https://www.w3.org/ns/solid/oidc-context.jsonld` I think it makes it the authoritative document.

Reply:
+1 on adding a namespaces section

Reply:
https://github.com/solid/authentication-panel/issues/170